src/yunohost/certificate.py

```diff
@@ -731,8 +731,11 @@ def _dns_ip_match_public_ip(public_ip, domain):
 
 def _domain_is_accessible_through_HTTP(ip, domain):
     try:
-        r = requests.head("http://" + ip, headers={"Host": domain})
-        # Check we got the ssowat header in the response
+        # Check HTTP reachability
+        requests.head("http://" + ip, headers={"Host": domain})
```

Comment:
Nope, because this checks the webserver is reachable on port 80, which is the one used by Let's Encrypt. There might be some sneaky cases where 443 is reachable correctly but 80 is not (e.g. the user didn't map correctly the port of his/her internet box :stuck_out_tongue:)